sthgetter/sthgetter.go

```diff
@@ -93,16 +91,16 @@ func getCheckStoreSTH(ctx context.Context, lc *client.LogClient, sv *ct.Signatur
 func checkSTH(sth *ct.SignedTreeHead, receivedAt time.Time, sv *ct.SignatureVerifier, l *ctlog.Log) []error {
 	var errs []error
 	// Check that the STH signature verifies.
-	log.Printf("%s: %s: verifying STH signature...", l.URL, logStr)
+	glog.Infof("%s: %s: verifying STH signature...", l.URL, logStr)
 	if err := sv.VerifySTHSignature(*sth); err != nil {
 		errs = append(errs, &SignatureVerificationError{Err: err})
-		log.Printf("%s: %s: STH signature verification failed", l.URL, logStr)
+		glog.Infof("%s: %s: STH signature verification failed", l.URL, logStr)
```

Comment:
I wonder whether these should be warnings?

Reply:
Yes, good idea.  Am also wondering whether some of the others further up should be errors... Have changed a couple, see what you think?

Reply:
LGTM.